pkg/api/customization/cluster/actionhandler.go

```diff
@@ -481,3 +483,38 @@ func (a ActionHandler) getKubeConfig(apiContext *types.APIContext, cluster *mgmt
 
 	return a.ClusterManager.KubeConfig(cluster.ID, token), nil
 }
+
+func (a ActionHandler) RotateCertificates(actionName string, action *types.Action, apiContext *types.APIContext) error {
+	cluster, err := a.ClusterClient.Get(apiContext.ID, metav1.GetOptions{})
```

Comment:
I think we should be using access.ByID to access cluster object, 
https://github.com/rancher/rancher/blob/master/pkg/api/customization/cluster/actionhandler.go#L72

Reply:
will fix